### PR TITLE
DeltaStation (Playtest Version)

### PR DESCRIPTION
**Pull request by Okand37**
Test merge PR for DeltaStation (PlayTest Version)

### CODE REVIEW

**Comment by ChangelingRain** [comment]
WOO!
It's always nice to see new, well-made maps.

**Comment by Anonmare** [comment]
Got an image of the completed station?

**Comment by kevinz000** [comment]
M-merging F-from M-master!~

**Comment by ExcessiveUseOfCobblestone** [comment]
is there a genetics section on the map? I think Mekhi has a stuttering disability.

File: _maps/deltastation.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+		#define MAP_PATH "map_files/DeltaStation"
+		#define MAP_FILE "DeltaPlaytest.dmm"
+		#define MAP_NAME "DeltaPlaytest"
+
+		#define MAP_TRANSITION_CONFIG DEFAULT_MAP_TRANSITION_CONFIG
+
+#elif !defined(MAP_OVERRIDE)
+
+	#warn a map has already been included, ignoring MetaStation.
```
Comment (Cyberboss) [inline _maps/deltastation.dm]: >MetaStation

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by GunHog** [comment]
Pictures please?

**Comment by Okand37** [comment]
As to those asking; I don't have a stitched picture setup because this was just a pr to testmerge it on the server for a round to get immediate bug reports on what needs fixing, adding, changing, etc. Not much use spending 10-20 minutes stitching a big picture together when you're bound to be editing and tweaking things all over the map then just have to do it again. A stitch picture will be included when I actually make a full PR for the completed map, however.

**Comment by Okand37** [comment]
Going to close this and put up one more updated version of a testmerge. Forgive me, still figuring out all the tips and tricks of git.

**Pull request closed without merging**